helpers/encodeCallData.ts

```diff
@@ -35,8 +35,11 @@ export function addGasLimit(callData: string, gasLimit: number): string {
 }
 
 export function addArgs(callData: string, args: any[]): string {
-    const numArgsHex = new BigNumber(args.length).toString(16).padStart(uint32ArgBytes, '0');
+    if (args.length == 0) {
+        return '00'
```

Comment:
So, if there are no arguments, there is no presence marker, and the number of arguments is 0. Sounds good.

Reply:
actually 00 means "no data presence marker"

Reply:
a const would have made it clearer :)

Reply:
added const